ocaml/sdk-gen/powershell/autogen/dune

```diff
@@ -1,34 +1,15 @@
 (rule
-  (targets LICENSE.txt)
```

Comment:
The copy to .txt was done as a courtesy to Windows users because the latter does not know what to do with files without extensions and needs an extra click to open the file. Personally I'm not too fussed, plus that it adds complexity, but might any users complain?

Reply:
my thought process was also that the PS SDK is now cross platform and so I followed the standard for no extenion in `LICENSE` (since Linux distros mostly assume it's a plain text file with no extension)

the real reason I changed this is that I like to follow the standard of `LICENSE` having no extension but honestly it's not that important.

If you prefer, I can revert this change, just let me know

Reply:
I'm ok to leave it for now, we can always do it later if anyone complains.